contracts/refs/OracleRef.sol

```diff
@@ -105,6 +105,11 @@ abstract contract OracleRef is IOracleRef, CoreRef {
         }
         require(valid, "OracleRef: oracle invalid");
 
+        // Invert the oracle price if necessary
```

Comment:
This makes sense to me and I think you're right. Afaik we have never deployed this contract with `doInvert = true` so would never have come across. 

I was concerned that doing the inversion here might result in an integer division problem when the price is >1e18 (e.g. invert = 1e18 / 2e18 = 0). However the Decimal library looks like it handles that by first scaling up by `1e18`. So all good

Reply:
I tried to use it as is, and it definitely did not work. Once I wrote the logic out and stepped through it, I realized this change had to be made for an inversion + scaling to work.